MAUI/Popup/popup-positioning.md

```diff
@@ -264,4 +264,300 @@ public class ViewModel : INotifyPropertyChanged
+      var contactName = new Label()
+      {
+          VerticalTextAlignment = TextAlignment.Center,
+          LineBreakMode = LineBreakMode.NoWrap,
+          FontSize = 14
+      };
+      contactName.SetBinding(Label.TextProperty, new Binding("ContactName"));
+      var contactType = new Image()
+      {
+          VerticalOptions = LayoutOptions.Center,
+          HorizontalOptions = LayoutOptions.End,
+          HeightRequest = 20,
+          WidthRequest = 20,
+          Margin = new Thickness(0, 0, 8, 0)
+      };
+      contactType.SetBinding(Image.SourceProperty, new Binding("ContactType"));
+      var stackLayout = new StackLayout()
+      {
+          BackgroundColor = Colors.LightGray,
+          HeightRequest = 1,                
+      };      
+      grid.Children.Add(contactImage);
+      Grid.SetColumn(contactImage, 0);
+      Grid.SetRow(contactImage, 0);
+      grid.Children.Add(contactName);
+      Grid.SetColumn(contactName, 1);
+      Grid.SetRow(contactName, 0);
+      grid.Children.Add(contactType);
+      Grid.SetColumn(contactType, 2);
+      Grid.SetRow(contactType, 0);
+      grid.Children.Add(stackLayout);
+      Grid.SetColumn(stackLayout, 0);
+      Grid.SetRow(stackLayout, 1);
+      Grid.SetColumnSpan(stackLayout, 3);
+      return grid;
+    });
+    viewModel = new ContactsViewModel();
+    listView.ItemsSource = viewModel.Items;
+    listView.ItemSize = 56;
+    listView.SelectionMode = Syncfusion.Maui.ListView.SelectionMode.None;
+    listView.ItemTapped += OnListViewItemTapped;
+    sfPopup = new SfPopup();
+    sfPopup.WidthRequest = 220;
+    sfPopup.HeightRequest = 120; ;
+    sfPopup.ShowCloseButton = true;
+    sfPopup.ContentTemplate = new DataTemplate(() =>
+    {
+      return new Label()
+      {
+        Text = "ListView item is tapped",
+        FontSize = 14,
+        TextColor = Colors.Black,
+        FontFamily = "Roboto"
+      };
+    });        
+    this.Content = listView;  
+  }  
+}
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void OnListViewItemTapped(object sender, Syncfusion.Maui.ListView.ItemTappedEventArgs e)
+{
+  sfPopup.Show();
+}
+{% endhighlight %}
+{% endtabs %}
+
+![ListView with .NET MAUI Popup](Images/popup-positioning/show-maui-popup-when-listview-item-tap.png)
+
+Download the entire source code from GitHub [here](https://github.com/SyncfusionExamples/how-to-show-a-.net-maui-popup-when-tap-.net-maui-listview-item).
+
+### Display popup when interacting with a switch
+
+[SfPopup.IsOpen](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_IsOpen) is a bindable property and hence can be binded to any property and based on its value the popup will open or close. In the below code example, we have binded the `IsOpen` property with the `IsToggled` property of the the switch and the popup will be opened or closed as the switch toggles.

```

Comment:
[SfPopup.IsOpen](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_IsOpen) is a bindable property and hence can be bonded to any property and based on its value the popup will open or close. In the following code example, we have bound the `IsOpen` property with the `IsToggled` property of the switch, and the popup will be opened or closed as the switch toggles.